tests/foreman/cli/test_activationkey.py

```diff
@@ -845,7 +845,7 @@ def test_associate_product_2(self):
 
     @run_only_on('sat')
     @stubbed()
-    def test_associate_product_3(self):
+    def test_positive_update_redhat_custom_products(self):
```

Comment:
I wonder... there's no `Red Hat custom product` but either `Red Hat` or `Custom`. Is this test valid?